Makefile

```diff
@@ -1,37 +1,27 @@
-DESTDIR :=
```

Comment:
`DESTDIR` is used in the [AUR-script](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=blugon).
I would like to keep it in for now. If I change that script I will change it here in another commit.

Reply:
In that case, would it make sense to write not only $(PREFIX) but $(DESTDIR)$(PREFIX) to the blugon.py and blugon.service in the build target?

Reply:
From https://www.gnu.org/prep/standards/html_node/DESTDIR.html:
> Prepending the variable DESTDIR to each target in this way provides for staged installs, where the installed files are not placed directly into their expected location but are instead copied into a temporary location (DESTDIR). However, installed files maintain their relative directory structure and any embedded file names will not be modified. 

This is what is going on. `DESTDIR` is used so that the package manager can track all files and then later on actually install them from the package.

`PREFIX` on the other hand is the path that it will actually get installed to.

Reply:
Done.